Tests/Rules/EnvironmentEntryTests.swift

```diff
@@ -271,4 +271,97 @@ final class EnvironmentEntryTests: XCTestCase {
         """
         testFormatting(for: input, rule: .environmentEntry, options: FormatOptions(swiftVersion: "6.0"))
     }
+
+    func testEntryMacroReplacementWhenDefaultValueIsNotComputed() {
+        let input = """
+        struct ScreenStyleEnvironmentKey: EnvironmentKey {
+            static var defaultValue: ScreenStyle = ScreenStyle()
```

Comment:
We should probably also have a test with `static let`

```suggestion
            static let defaultValue: ScreenStyle = ScreenStyle()
```

Reply:
I thought I added the test, let me make sure it is included.